tests/cli/common.sh

```diff
@@ -1,9 +1,46 @@
-# this is not executable, but helper functions for the other scripts
+# This is not executable, but helper functions for the other scripts
+
+# XXX XXX XXX XXX XXX
+# The following global variables must be defined before calling common functions: 
+# SERVER_EXE=foobar       # Server binary name
+# CLIENT_EXE=foobarcli    # Client binary name
+# ACCOUNTS=(foo bar)      # List of accounts for initialization 
+# RICH=${ACCOUNTS[0]}     # Account to assign genesis balance
+
+# XXX Ex Usage: quickSetup $WORK_NAME $CHAIN_ID 
+# Desc: Start the program, use with shunit2 OneTimeSetUp()
+quickSetup() {
+  # These are passed in as args
+  BASE_DIR=$HOME/$1
+  CHAIN_ID=$2
+
+  rm -rf $BASE_DIR 2>/dev/null
+  mkdir -p $BASE_DIR
+
+  # Set up client - make sure you use the proper prefix if you set
+  #   a custom CLIENT_EXE
+  export BC_HOME=${BASE_DIR}/client
+  prepareClient
+
+  # start basecoin server (with counter)
+  initServer $BASE_DIR $CHAIN_ID
+  if [ $? != 0 ]; then return 1; fi
+
+  initClient $CHAIN_ID
+  if [ $? != 0 ]; then return 1; fi
+
+  printf "...Testing may begin!\n\n\n"
+}
+
+# XXX Ex Usage: quickTearDown
+# Desc: close the test server, use with shunit2 OneTimeTearDown()
+quickTearDown() {
+  printf "\n\nstopping $SERVER_EXE test server..."
```

Comment:
Wondering why printf over echo, but saw some links.... guess I need to update all the scripts now... (or maybe you did ;)